reST/makeTable.py

```diff
@@ -113,9 +113,9 @@ def toRSTtable(rows, header=True, vdelim="  ", padding=2, justify='center'):
     return "\n".join(result)
 
 if __name__ == '__main__':
-    labels = ('IOMode','Peut écrire ?','Peut lire ?','Position de départ', 'Notes')
+    labels = (u'IOMode',u'Peut écrire ?',u'Peut lire ?',u'Position de départ', u'Notes')
     data = \
-    '''ReadMode,Oui,Non,Début de fichier,Le fichier doit déjà exister
+    u'''ReadMode,Oui,Non,Début de fichier,Le fichier doit déjà exister
```

Comment:
I had a look at what this file does, but it didn't work with python 2.7 without adding a few unicode prefixes. If we want to support this it should be ported to python 3, but that requires more work. Since I took the time to figure out what went wrong in python 2.7 I thought I'd offer to have the fixes included in master as well so if someone wants to port it to python 3 they can start form a working version.